lefthook.yml

```diff
@@ -0,0 +1,40 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format non-markdown
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --write --cache {staged_files}
+      stage_fixed: true
+
+    - name: Fix markdown
+      glob: "*.md"
+      group:
+        parallel: true
+        jobs:
+          - name: Markdownlint
+            run: yarn markdownlint-cli2 --fix {staged_files}
+            stage_fixed: true
+          - name: Check URLs per locale
+            run: yarn node ./scripts/check-url-locale.js --fix {staged_files}
+            stage_fixed: true
```

Comment:
@yin1999 - this script hangs for me if I try to trigger it with this in `files/ja/web/api/fetch_api/using_fetch/index.md`:

```diff
+ [フェッチ API](/es/docs/Web/API/Fetch_API) 
- [フェッチ API](/ja/docs/Web/API/Fetch_API) 
```

What do you think about leaving this out for now until we get it running? 

```suggestion
```

Aside from being able to run this script independently of pre-commit, this could be the type of thing that's built into rari directly in future. See https://github.com/mdn/rari/pull/158

Reply:
Hey, @bsmth. I tested this on my device and it seems to work fine without hangs. Did you test with the commit that only contained changes to this link? This may produce an empty commit (requiring you to fill in the commit message interactively).

Reply:
Actually the problem is the ora spinner which has some TTY interference. I've removed it in https://github.com/mdn/translated-content/pull/26482/commits/f69fa8535a0dec5f002e4147bf7ee536586108d9 which works a lot better, any thoughts on this?

Reply:
Thanks for your thorough investigation, I have no objection to removing the ora spinner from the script. I think it would be better if rari could integrate these functions directly in the future 😄

Reply:
>I think it would be better if rari could integrate these functions directly in the future 

You're very welcome and I totally agree.